oaknational/Oak-Web-Application
# chore: lesson listing 5 LESQ-106

## Description

- Connects lesson listing 5 
- It should now have synthetic programme pages

## How to test
/beta/teachers-2023/programmes/maths-secondary-ks4-foundation/units/algebraic-manipulation/lessons 
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
